meta-lxatac-bsp/recipes-core/images/emmc-boot-image/lxatac/genimage.config

```diff
@@ -18,7 +18,7 @@ image @IMAGE@fip {
     fip {
         fw-config = "stm32mp157c-lxa-tac-fw-config.dtb"
         hw-config = "stm32mp157c-lxa-tac-fw-config.dtb"
-        nt-fw = "barebox-stm32mp15xc-lxa-tac.img"
+        nt-fw = "barebox-stm32mp15xc-lxa-tac.stm32.img"
```

Comment:
I think this should go away with the to-be-upstreamed OE-core recipe.

Reply:
It just spins the filename extension carousel one step further by changing the extension to `barebox-stm32mp15xc-lxa-tac.stm32`.

Reply:
that's the correct name though. `.img` is misleading as the `.stm32` images only work by chance for your use case: The stm32 header in front is no longer parsed by TF-A and is directly executed. I think we are lucky that the header evaluates to some arithmetic instructions that don't crash.

Reply:
Huh, I don't like the sound of that. What would be the proper solution here?
Is it something we should squeeze in with this PR or address in an issue?

Reply:
deal!

Reply:
I couldn't rebuild the umpf tag, so here goes my change. Please test and squash if it works:
```
commit 4d6c3ef497f7d39c15cbd3923a1c038026c2653a (HEAD)
Author:     Ahmad Fatoum <a.fatoum@pengutronix.de>
AuthorDate: Thu Apr 4 22:16:35 2024 +0200
Commit:     Ahmad Fatoum <a.fatoum@pengutronix.de>
CommitDate: Thu Apr 4 22:18:09 2024 +0200

    fixup! ARM: stm32mp: add Linux Automation TAC board
    
    ARM: stm32mp: lxa-tac: remove STM32 header from bootloader binary
    
    The STM32 header is what's parsed by the BootROM of the STM32MP1. TF-A
    used to expect the same header for the BL33 binary it loads, but it no
    longer does and instead expects a FIP image that contains amongs others
    the BL33 binary as directly executable binary without header and a
    device tree that it passes to it.
    
    For use with FIP, barebox provides a barebox-stm32mp-generic-bl33.img
    image that is not board-specific. We don't use that for the LXA TAC as
    we don't want to settle on a single device tree and instead want to
    select the correct device tree at runtime from multiple that are linked
    into barebox.
    
    Instead we continue to use the .stm32 image, which still has the STM32
    header, which is now directly executed. This didn't lead to problems so
    far, because the header happens to translate to valid ARM arithmetic
    instructions that don't break anything, but we shouldn't rely on that.
    
    Fix this by generating a plain image instead without header.
    
    Signed-off-by: Ahmad Fatoum <a.fatoum@pengutronix.de>

diff --git a/images/Makefile.stm32mp b/images/Makefile.stm32mp
index e8c7adc9d8f6..80f6b9dfc8f5 100644
--- a/images/Makefile.stm32mp
+++ b/images/Makefile.stm32mp
@@ -34,7 +34,10 @@ $(call build_stm32mp_image, CONFIG_MACH_STM32MP15XX_DKX, start_stm32mp15xx_dkx,
 $(call build_stm32mp_image, CONFIG_MACH_STM32MP15X_EV1, start_stm32mp15x_ev1, stm32mp15x-ev1)
 
 $(call build_stm32mp_image, CONFIG_MACH_LXA_MC1, start_stm32mp157c_lxa_mc1, stm32mp157c-lxa-mc1)
-$(call build_stm32mp_image, CONFIG_MACH_LXA_TAC, start_stm32mp15xc_lxa_tac, stm32mp15xc-lxa-tac)
+
+pblb-$(CONFIG_MACH_LXA_TAC) += start_stm32mp15xc_lxa_tac
+FILE_barebox-stm32mp15xc-lxa-tac.img  = start_stm32mp15xc_lxa_tac.pblb
+image-$(CONFIG_MACH_LXA_TAC) += barebox-stm32mp15xc-lxa-tac.img
 
 $(call build_stm32mp_image, CONFIG_MACH_PROTONIC_STM32MP1, start_prtt1a, prtt1a)
 $(call build_stm32mp_image, CONFIG_MACH_PROTONIC_STM32MP1, start_prtt1s, prtt1s)
```

Reply:
I've applied the change and tested it now and it seems to work fine.

Reply:
I guess that means this PR is ok to merge then.